### PR TITLE
Revert "AlignDoubleArrowFixer - fix handling of nested array"

### LINKED ISSUE #1450: align_double_arrow looks weird
Since the last update ``align_double_arrow``looks weird

```bash
oskar.stark:/Volumes/development/workspaces/SonataBlockBundle (master)$ make cs
./vendor/bin/php-cs-fixer fix --verbose
Loaded config from "/Volumes/development/workspaces/SonataBlockBundle/.php_cs"
.................FF...........................................................................................
Legend: ?-unknown, I-invalid file syntax, file ignored, .-no changes, F-fixed, E-error
   1) Block/Service/MenuBlockService.php (align_double_arrow)
   2) Block/Service/RssBlockService.php (align_double_arrow)
Fixed all files in 23.432 seconds, 9.500 MB memory used
make: *** [cs] Error 1
oskar.stark:/Volumes/development/workspaces/SonataBlockBundle (master)$ git diff
diff --git a/Block/Service/MenuBlockService.php b/Block/Service/MenuBlockService.php
index 6cd0965..2c88923 100644
--- a/Block/Service/MenuBlockService.php
+++ b/Block/Service/MenuBlockService.php
@@ -133,16 +133,16 @@ class MenuBlockService extends BaseBlockService
     protected function getFormSettingsKeys()
     {
         return array(
-            array('title', 'text', array('required' => false)),
-            array('cache_policy', 'choice', array('choices' => array('public', 'private'))),
-            array('menu_name', 'choice', array('choices' => $this->menus, 'required' => false)),
+            array('title', 'text', array('required'           => false)),
+            array('cache_policy', 'choice', array('choices'   => array('public', 'private'))),
+            array('menu_name', 'choice', array('choices'      => $this->menus, 'required' => false)),
             array('safe_labels', 'checkbox', array('required' => false)),
-            array('current_class', 'text', array('required' => false)),
-            array('first_class', 'text', array('required' => false)),
-            array('last_class', 'text', array('required' => false)),
-            array('menu_class', 'text', array('required' => false)),
-            array('children_class', 'text', array('required' => false)),
-            array('menu_template', 'text', array('required' => false)),
+            array('current_class', 'text', array('required'   => false)),
+            array('first_class', 'text', array('required'     => false)),
+            array('last_class', 'text', array('required'      => false)),
+            array('menu_class', 'text', array('required'      => false)),
+            array('children_class', 'text', array('required'  => false)),
+            array('menu_template', 'text', array('required'   => false)),
         );
     }

diff --git a/Block/Service/RssBlockService.php b/Block/Service/RssBlockService.php
index 80e906e..be76f09 100644
--- a/Block/Service/RssBlockService.php
+++ b/Block/Service/RssBlockService.php
@@ -51,7 +51,7 @@ class RssBlockService extends BaseBlockService
     {
         $formMapper->add('settings', 'sonata_type_immutable_array', array(
             'keys' => array(
-                array('url', 'url', array('required' => false)),
+                array('url', 'url', array('required'    => false)),
                 array('title', 'text', array('required' => false)),
             ),
         ));
oskar.stark:/Volumes/development/workspaces/SonataBlockBundle (master)$ vendor/bin/php-cs-fixer --version
PHP CS Fixer version 1.10.1 by Fabien Potencier
oskar.stark:/Volumes/development/workspaces/SonataBlockBundle (master)$
```

### PR DESCRIPTION
**Pull request by Soullivaneuh**
This reverts commit f7777cf707b840edb1101eeec43cd6ace5749128.

Fixes #1450.

cc @SpacePossum

### CODE REVIEW

**Comment by SpacePossum** [comment]
:+1: 
not all reverts are needed, but I get it :)

(it would be nice to have a test added that proves the revert fixes the issue. It would also prevent breaking (in that way) again in the future,)

**Comment by Soullivaneuh** [comment]
> it would be nice to have a test added that proves the revert fixes the issue

You have it on #1450. :-)

**Comment by Soullivaneuh** [comment]
Ping @keradus this is quite urgent IMO.

**Comment by keradus** [comment]
No tests to prove the behavior was added, but that was already said by @SpacePossum

File: Symfony/CS/Fixer/Contrib/AlignDoubleArrowFixer.php [hunk 1]
```diff
@@ -102,7 +110,7 @@ private function injectAlignmentPlaceholders(Tokens $tokens, $startAt, $endAt)
                 continue;
             }
 
-            if ($tokens->isShortArray($index)) {
+            if ($token->equals('[')) {
```
Comment (keradus) [inline Symfony/CS/Fixer/Contrib/AlignDoubleArrowFixer.php]: this change is bad

File: Symfony/CS/Fixer/Contrib/AlignDoubleArrowFixer.php [hunk 2]
```diff
@@ -79,8 +79,16 @@ public function getPriority()
      *
      * @return array($code, $context_counter)
      */
-    private function injectAlignmentPlaceholders(Tokens $tokens, $startAt, $endAt)
+    private function injectAlignmentPlaceholders(Tokens $tokens, $startAt = null, $endAt = null)
```
Comment (keradus) [inline Symfony/CS/Fixer/Contrib/AlignDoubleArrowFixer.php]: this change is not needed

**Comment by Soullivaneuh** [comment]
> No tests to prove the behavior were added, but that was already said by @SpacePossum

Will work on it soon.

I don't understand your code comment about bad changes. I just reverted the commit.

**Comment by keradus** [comment]
Not all work need to be reverted, then why to throw it away?

**Comment by Soullivaneuh** [comment]
> Not all work need to be reverted, then why to throw it away?

@SpacePossum said that he doesn't have time to fix it for the moment on proposed to revert the PR.

**Comment by keradus** [comment]
And I said that not all of the work need to be reverted blindly.

**Comment by Soullivaneuh** [comment]
@SpacePossum Could you tell me which changes need to be reverted?

**Comment by keradus** [comment]
If you won't believe the comments simply add missing tests and it will be clear ;)

**Comment by Soullivaneuh** [comment]
@keradus @SpacePossum Here is the proof: https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1454

**Comment by keradus** [comment]
the proof should be in THIS PR to prove that this change will fix it.

**Comment by Soullivaneuh** [comment]
@keradus added.

**Comment by keradus** [comment]
Proof fails with and without revert.

**Comment by Soullivaneuh** [comment]
Ok. No time to fix it for the moment.

@SpacePossum I let your the liberty to cherry-pick my proof to see where is the problem. :+1:

**Comment by keradus** [comment]
Closing PR as the revert is not a fix.
If anyone want to work on the fix please start from #1455 and I will be happy to merge it ;)

**Comment by Soullivaneuh** [comment]
@keradus I manually tested the revert with my proof tests.

This fix one of them. I think the other (`new DummyObject`) is unrelated.

Possibly not a bug, or one introduced long time ago.

Will remove this one for the moment. What do you think about it?

**Pull request closed without merging**